plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -47,17 +47,25 @@ def __init__(self, configurations: Dict[str, Dict]):
         self._check_config_equal(
             key="peft.quantization.auto_gptq.from_quantized", value=True
         )
+        self.use_external_lib = use_external_lib and importlib.util.find_spec("autogptq") is not None
```

Comment:
just do an importlib check if `use_external_lib` is True, and raise if not found